src/TowseyLibrary/MatrixTools.cs

```diff
@@ -584,7 +584,7 @@ public static double[] Matrix2Array(double[,] matrix)
                 for (int j = 0; j < cols; j++)
                 {
                     //convert decibels to power
-                    double power = Math.Exp(m[i, j] / 10 * Math.Log(10));
+                    double power = Math.Exp(m[i, j] / 10);
```

Comment:
Ok this seems like a subtle but far reaching change. Can you document the reason for the change?